utils/common/src/main/java/org/apache/brooklyn/util/exceptions/RuntimeInterruptedException.java

```diff
@@ -33,18 +33,14 @@
 
     private static final long serialVersionUID = 915050245927866175L;
 
-    public RuntimeInterruptedException(InterruptedException cause) {
+    public RuntimeInterruptedException(Exception cause) {
```

Comment:
Slightly uncomfortable about it taking `Exception`, if we intend that it is only ever constructed with an `InterruptedException` or another `RuntimeInterruptedException`.